src/screens/Customers/CustomerMain.tsx

```diff
@@ -128,7 +130,15 @@ function CustomerMain(props: CustomerMainProps) {
 
   const getCustomers = () => {
     if (searchValue !== '') {
-      const searchedCustomersArray = allCustomersTrie.get(searchValue);
+      let searchedCustomersArray;
```

Comment:
The more I think about it, the more I think we can just use `useMemo` somewhere here to speed up the slow tabbing / calculation processes. I'm not sure where though and we have more important to fill out, so it might be wise to just come back to put a TODO explaining that this could probably be sped up using React's useMemo, provide a link, and come back later.